labs.yml

```diff
@@ -59,6 +59,9 @@ commands:
 
   - name: ensure-assessment-run
     description: ensure the assessment job was run on a workspace
+    flags:
+      - name: run-as-collection
```

Comment:
@HariGS-DB : Did you verify the command with this flag? I want a boolean flag in lsql too, but expect this to be a flag requiring an argument

Reply:
I suspect this does not work as expected, see https://github.com/databrickslabs/lsql/pull/235